pages/course/[courseId]/index.tsx

```diff
@@ -27,22 +35,24 @@ export default function Details() {
   const course = coursesById[0];
 
   //TODO: change later, when implementing the services into this side
-  const chapters = [
-    "Chapter 1",
-    "Chapter 2",
-    "Chapter 3",
-    "Chapter 4",
-    "Chapter 5",
-    "Chapter 6",
-    "Chapter 7",
-    "Chapter 8",
-    "Chapter 9",
-    "Chapter 10",
-  ];
+  const chapters = course.chapters;
   const knowledge = 88;
   const understanding = 50;
   const analyses = 40;
   const usage = 22;
+  let viewablechapters = [];
+
+  for (let index = 0; index < chapters.length; index++) {
+    const element = chapters.at(index);
+    const start = new Date(element?.startDate);
+    const end = new Date(element?.endDate);
+    if (
+      start.toISOString() <= currentDate.toISOString() &&
+      end.toISOString() >= currentDate.toISOString()
```

Comment:
Comparing them directly (`start <= currentdate`) should be better than string comparison.

Reply:
I based it on following [article](https://programmingwithswift.com/how-to-compare-dates-with-typescript/). There they tested it with string and with normal date comparison, but the normal comparison did not work correctly. But if this article is outdate I can change it.

Reply:
I guess I'd prefer `.getTime()`, but lets get this merged.l as it's personal preference.